packages/context/src/keys.ts

```diff
@@ -29,6 +29,11 @@ export namespace ContextTags {
    */
   export const KEY = 'key';
 
+  /**
+   * Binding tag to associate a configuration binding with the target binding key
+   */
+  export const CONFIGURATION_OF = 'configurationOf';
```

Comment:
`CONFIGURATION_OF` looks a bit weird to me, have you considered any alternative names? E.g. `CONFIGURATION_FOR`?

(Nitpick - feel free to ignore.)